engines/dfc_provider/app/services/dfc_provider/authorization_control.rb

```diff
@@ -8,24 +8,55 @@ def initialize(access_token)
       @access_token = access_token
     end
 
+    def safe_process
+      process
+    rescue JWT::VerificationError, JWT::ExpiredSignature, JWT::ImmatureSignature => e
+      # To help development debugging
+      raise e if Rails.env.development?
+
+      nil
+    end
+
```

Comment:
The idea here was not to send the error message to the end user, for security reason, but maybe it is not an issue.